app/parse.py

```diff
@@ -1,12 +1,62 @@
+import requests
+from bs4 import BeautifulSoup
+
 from dataclasses import dataclass
 
 
+BASE_URL = "https://mate.academy/"
+
+
 @dataclass
 class Course:
     name: str
     short_description: str
     duration: str
+    count_topics: int = None
```

Comment:
The `count_topics` attribute is initialized with `None`, which is fine, but ensure that `get_count_of_topics` always returns an integer. Consider adding error handling in `get_count_of_topics` to manage cases where the element might not exist or contain non-numeric text.